lua/gk.lua

```diff
@@ -15,8 +15,50 @@ return function (net_conf, numa_table)
 	local ggu_lcore = table.remove(gk_lcores)
 	gatekeeper.gk_assign_lcores(gk_conf, gk_lcores)
 
+	gk_conf.max_num_ipv4_rules = 1024
+	gk_conf.num_ipv4_tbl8s = 256
+	gk_conf.max_num_ipv6_rules = 1024
+	gk_conf.num_ipv6_tbl8s = 65536
+
+	local fib_entries = {
+		{
+			["ip_addr"] = "187.73.32.0",
+			["prefix_len"] = 30,
+			["action"] = gatekeeper.c.GK_FWD_GRANTOR,
+			["grantor"] = "10.0.0.3",
+		},
+
+		{
+			["ip_addr"] = "187.73.33.0",
+			["prefix_len"] = 30,
+			["action"] = gatekeeper.c.GK_DROP,
+		},
+	}
+
+	local grantors = {
+		{
+			["gw_id"] = -1,
+			["grantor"] = "10.0.0.3",
+		},
+
+		{
+			["gw_id"] = 0,
+			["grantor"] = "3ffe:2501:200:3::1",
+		},
+	}
+
+	local gateways = {
```

Comment:
To add a fib entry, it will first add the entries for gateway, since they may be referenced by entries for grantor servers, then the entreis for grantor servers will be referenced by entries for IP prefixes. So, I maintained the gateway entries as a separate list. Later, the administrators can easily add/delete new entries for gateways by calling its corresponding interfaces. Also, I made another interface for updating the fib entries for IP prefixes.